cartridge/auth-backend.lua

```diff
@@ -86,6 +120,13 @@ end
 local function get_user(username)
     checks('string')
 
+    if ADMIN_USER.enabled and username == ADMIN_USER.username then
+        return {
+            username = ADMIN_USER.username,
+            fullname = ADMIN_USER.fullname,
+        }
```

Comment:
nit: I guess it could be moved into separate function. Such table is used in several places.